js/widgets/temperament.js

```diff
@@ -782,7 +782,12 @@ function TemperamentWidget() {
         this.tempRatios = [];
 
         divAppend.addEventListener("click", function (event) {
-            that.performEqualEdit(event);
+            try{
+                that.performEqualEdit(event);
+            } catch{
+                this.activity.errorMsg(_("Number of divisions is too large."));
```

Comment:
Could you please change it to "The number of divisions is too large." ?

Reply:
sure.